NetworkingServiceKit/Classes/Networking/NetworkManager.swift

```diff
@@ -176,6 +176,36 @@ public enum HTTPMethod: Int32 {
     }
 }
 
+/// Custom type of cache policy
+@objc
+public enum CacheResponsePolicyType : UInt {
+    
+    /// Cache based on HTTP-Header: Cache-Control
+    case cacheControlBased
+    
+    /// Force Cache this response
+    case forceCache
```

Comment:
Could we add an additional policy type that explicitly overrides the cache? I think that's clearer than getting the same effect with a forceCache with a maxAge of 0